airbyte-bootloader/src/main/java/io/airbyte/bootloader/BootloaderApp.java

```diff
@@ -168,7 +168,10 @@ private static void createWorkspaceIfNoneExists(final ConfigRepository configRep
     configRepository.writeStandardWorkspace(workspace);
   }
 
-  private static void assertNonBreakingMigration(JobPersistence jobPersistence, AirbyteVersion airbyteVersion) throws IOException {
+  private static void assertNonBreakingMigration(
```

Comment:
we usually have the first arg on the same line as method declaration.

Reply:
Fixed